### PR TITLE
Resolve DeprecationWarnings under python 36

### PR DESCRIPTION
**Pull request by pvinci**

### CODE REVIEW

File: testtools/content.py [hunk 1]
```diff
@@ -102,7 +102,7 @@ def iter_text(self):
         no charset parameter is present in the MIME type. (This is somewhat
         arbitrary, but consistent with RFC2617 3.7.1).
 
-        :raises ValueError: If the content type is not text/\\*.
+        :raises ValueError: If the content type is not text *.
```
Comment (jelmer) [inline testtools/content.py]: Perhaps "text/*" ? Replacing it with a space makes it unparseable to me.

**Comment by ssbarnea** [comment]
I think that any core can fix this PR. Anyone?

File: testtools/content.py (hunk 1 above)
Comment (mtreinish) [inline testtools/content.py:105]:
```suggestion
        :raises ValueError: If the content type is not "text/*".
```

**Review by ssbarnea — APPROVED** [review]
LGTM

**Review by mtreinish — APPROVED** [review]

**Pull request merged**